### ISSUE TITLE
local installs without aliases throw exception on Windows

### ISSUE THREAD

**Issue by bobisageek** [babashka/bbin#37]
**pre-emptive apology**
I don't mean to be the "but what about Windows?" guy, but...

**problem statement**
Installing a local clj on Windows throws:
```
>bbin version
bbin 0.1.3

>bbin install hellocal.clj
{:coords {:bbin/url "file://C:\\testing_stuff\\hellocal.clj"}}
----- Error --------------------------------------------------------------------
Type:     java.lang.IllegalArgumentException
Message:  C:\testing-stuff\hellocal is not a relative path
Location: C:\path\to\bbin:398:38
...
398:         script-file (fs/canonicalize (fs/file (util/bin-dir cli-opts) script-name)
                                          ^--- C:\testing-stuff\hellocal is not a relative path
... (longer details elided for brevity)
```
but works if an alias is supplied:
```
>bbin install hellocal.clj --as hellocal
{:coords {:bbin/url "file://C:\\testing_stuff\\hellocal.clj"}}
```

**analysis**
- the error and the fact that it works with an alias suggests that `script-name` is problematic
- `script-name` comes from `file-path->script-name`, which is splitting the path on '/', which doesn't produce the desired result on a Windows canonicalized path
- I haven't tested on a local jar yet, but I assume the same symptom and root cause would pop up there

I haven't started on a PR yet, but a little thinking about it raised another question in my head:
The current `file-path->script-name` takes the first split of the file name on `.`. Would it be desirable to drop the extension instead? By which I mean 'foo.bar.clj' would be installed as 'foo.bar'. I realize this is _very_ unlikely to make a difference because of namespace names and what not, but I was considering using `fs/strip-ext` in the PR, which would _technically_ be different in the edge case.

**Comment by rads** [comment]
No need to apologize, that’s my bad. I created an issue to get Windows in CI so these things can be caught before release.

https://github.com/babashka/bbin/issues/41

**Comment by rads** [comment]
Resolved in #38.
